vs/client/src/views/projectExplorer/ileObjectTreeItem.ts

```diff
@@ -0,0 +1,31 @@
+import { ILEObject } from "@ibm/sourceorbit/dist/src/targets";
+import { ProjectExplorerTreeItem } from "@ibm/vscode-ibmi-projectexplorer-types/views/projectExplorer/projectExplorerTreeItem";
+import { ThemeIcon, TreeItem, TreeItemCollapsibleState, Uri, WorkspaceFolder } from "vscode";
+import { LanguageClientManager } from '../../languageClientManager';
+import { TypeIcons } from '../utils';
+import { NoticeTreeItem } from './noticeTreeItem';
+import path = require('path');
+
+export class ILEObjectTreeItem extends TreeItem implements ProjectExplorerTreeItem {
+	constructor(public workspaceFolder: WorkspaceFolder, private ileObject: ILEObject, canExpand = false) {
+		super(`${ileObject.systemName}.${ileObject.type}`, canExpand ? TreeItemCollapsibleState.Collapsed : TreeItemCollapsibleState.None);
+		// const logs = TargetsManager.getLogs(workspaceFolder, ileObject);
```

Comment:
@worksofliam Are these logs something that we should render? Maybe in the tooltip?